settings.gradle

```diff
@@ -9,6 +9,7 @@ include ':redwood-layout-schema'
 include ':redwood-layout-schema:compose'
 include ':redwood-layout-schema:layout-modifiers'
 include ':redwood-layout-schema:widget'
+include ':redwood-layout-composeui'
```

Comment:
Move up below `-api` to maintain alphabetization (is that a word?)

Reply:
It [checks out](https://www.merriam-webster.com/dictionary/alphabetization#:~:text=Definition%20of%20alphabetization,arranged%20series%2C%20list%2C%20or%20file)!